frontends/qt/Makefile

```diff
@@ -36,7 +36,8 @@ linux:
 	cd build/linux && fpm --after-install ../../resources/deb-afterinstall.sh -s dir -t deb -n bitbox -v 4.13.0 -C ../tmp-deb/
 	cd build/linux && fpm --after-install ../../resources/deb-afterinstall.sh -s dir -t rpm -n bitbox -v 4.13.0 -C ../tmp-deb/
 	# create AppImage
-	sed 's+/usr/share/pixmaps/bitbox.png+default+' resources/linux/usr/share/applications/bitbox.desktop > build/linux-tmp/bitbox.desktop
+	cp resources/linux/usr/share/applications/bitbox.desktop build/linux-tmp
+	cp resources/linux/usr/share/icons/hicolor/128x128/apps/bitbox.png build/linux-tmp
```

Comment:
Tested and looks great for the AppImage.

The .deb/.rpm packages do not contain the bitbox.png though for the Qt window logo. You should move this line to before the `fpm` calls above.

Reply:
sorry not move, but have the same for `../tmp-deb`

Reply:
fixed